packages/app-accounts/src/modals/Create.tsx

```diff
@@ -173,8 +174,12 @@ function Create ({ className, onClose, onStatusChange, seed: propsSeed, type: pr
     { value: 'raw', text: t('Raw seed') }
   ), [isDevelopment, t]);
 
-  const _onChangePass = (password: string): void =>
-    setPassword({ isPassValid: keyring.isPassValid(password), password });
+  const _onChangePass = (password: string): void => {
+    setPassword({isPassValid: keyring.isPassValid(password), password});
```

Comment:
The formatting of this line doesn't look consistent with the other lines.

Reply:
done